src/ZEO/tests/testZEO.py

```diff
@@ -262,6 +259,15 @@ def setUp(self):
             )
         self._storage.registerDB(DummyDB())
 
+    # _new_storage_client opens another ClientStorage to the same storage server
+    # self._storage is connected to. It is used by both ZEO and ZODB tests.
+    def _new_storage_client(self):
+        client = ZEO.ClientStorage.ClientStorage(
+            self._storage._addr, wait=1, **self._client_options())
+        client = self._wrap_client(client)
+        client.registerDB(DummyDB())
+        return client
```

Comment:
Should the storage really already register with a DB? In some situation, we might want an `MVCCAdapter` to be registered.

Reply:
@d-maurer, thanks for feedback. I've thought about that too, and my logic here is:

- the original code being factored-out had that `registerDB(DummyDB())`. There is also the same `registerDB(DummyDB())` in `GenericTestBase.setUp` which prepares `._storage` client for practically most of the tests:

https://github.com/zopefoundation/ZEO/blob/707316c49984ffbf8f2347611de5abdc3230dd3f/src/ZEO/tests/testZEO.py#L236-L263

- the `DB.registerDB`, when called for the second time, forgets what was registered first. This way when/if MVCCAdapter registers into ClientStorage, the registration of DummyDB is dropped.

The code in `_new_storage_client` is this OK, and is in line with the way `ClientStorage` for main `._storage` is prepared.